projects/plugins/jetpack/extensions/blocks/tiled-gallery/layout/index.js

```diff
@@ -47,6 +48,9 @@ export default class Layout extends Component {
 
 		const { src, srcSet } = photonizedImgProps( img, { layoutStyle } );
 
+		const origUrlObject = new URL( img.url );
+		const isRequestAgainstBlog = true; // TODO How do I know if the image is on my own blog?
+		const proxySiteIdOrSlug = 'flarypod.jurassic.tube'; // TODO How do I find my site_id?
```

Comment:
`Jetpack_Editor_Initial_State` has a `wpcomBlogId` that could be [helpful](https://github.com/Automattic/jetpack/blob/12c1ec934857a0c0fa2afe2ddecbddb7cdbf2e4e/projects/plugins/jetpack/class.jetpack-gutenberg.php#L737).